toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/WorldScaleSceneViewScope.kt

```diff
@@ -0,0 +1,99 @@
+/*
+ *
+ *  Copyright 2024 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package com.arcgismaps.toolkit.ar
+
+import androidx.compose.foundation.layout.BoxScope
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.Offset
+import com.arcgismaps.geometry.Point
+import com.arcgismaps.mapping.GeoElement
+import com.arcgismaps.toolkit.geoviewcompose.SceneViewScope
+import com.arcgismaps.toolkit.geoviewcompose.theme.CalloutColors
+import com.arcgismaps.toolkit.geoviewcompose.theme.CalloutDefaults
+import com.arcgismaps.toolkit.geoviewcompose.theme.CalloutShapes
+
+/**
+ * The receiver class of the [WorldScaleSceneView] content lambda.
+ *
+ * @since 200.6.0
```

Comment:
```suggestion
 * @since 200.7.0
```

Also in other places in this file...